cmd/tether/attach.go

```diff
@@ -269,6 +269,13 @@ func (t *attachServerSSH) stop() error {
 func backchannel(ctx context.Context, conn net.Conn) error {
 	defer trace.End(trace.Begin("establish tether backchannel"))
 
+	// A series of unfortunate events can lead calling backchannel with nil when we run unit tests.
+	// https://github.com/vmware/vic/pull/5327#issuecomment-305619860
+	// This check is here to handle that
+	if conn == nil {
```

Comment:
How about checking that before calling backchannel? This way we can check for t.testing too otherwise we need to find another way to pass that info to backchannel